src/azure-cli/azure/cli/command_modules/acs/custom.py

```diff
@@ -66,15 +66,17 @@
 from azure.mgmt.containerservice.v2019_11_01.models import ManagedClusterAgentPoolProfile
 from azure.mgmt.containerservice.v2019_11_01.models import AgentPool
 
-from azure.mgmt.containerservice.v2019_09_30_preview.models import OpenShiftManagedClusterAgentPoolProfile
-from azure.mgmt.containerservice.v2019_09_30_preview.models import OpenShiftAgentPoolProfileRole
-from azure.mgmt.containerservice.v2019_09_30_preview.models import OpenShiftManagedClusterIdentityProvider
-from azure.mgmt.containerservice.v2019_09_30_preview.models import OpenShiftManagedClusterAADIdentityProvider
-from azure.mgmt.containerservice.v2019_09_30_preview.models import OpenShiftManagedCluster
-from azure.mgmt.containerservice.v2019_09_30_preview.models import OpenShiftRouterProfile
-from azure.mgmt.containerservice.v2019_09_30_preview.models import OpenShiftManagedClusterAuthProfile
-from azure.mgmt.containerservice.v2019_09_30_preview.models import NetworkProfile
-from azure.mgmt.containerservice.v2019_09_30_preview.models import OpenShiftManagedClusterMonitorProfile
+from azure.mgmt.containerservice.v2019_10_27_preview.models import OpenShiftManagedClusterAgentPoolProfile
+from azure.mgmt.containerservice.v2019_10_27_preview.models import OpenShiftManagedClusterMasterPoolProfile
+from azure.mgmt.containerservice.v2019_10_27_preview.models import OpenShiftAgentPoolProfileRole
+from azure.mgmt.containerservice.v2019_10_27_preview.models import OpenShiftManagedClusterIdentityProvider
+from azure.mgmt.containerservice.v2019_10_27_preview.models import OpenShiftManagedClusterAADIdentityProvider
+from azure.mgmt.containerservice.v2019_10_27_preview.models import OpenShiftManagedCluster
+from azure.mgmt.containerservice.v2019_10_27_preview.models import OpenShiftRouterProfile
+from azure.mgmt.containerservice.v2019_10_27_preview.models import OpenShiftManagedClusterAuthProfile
+from azure.mgmt.containerservice.v2019_10_27_preview.models import NetworkProfile
+from azure.mgmt.containerservice.v2019_10_27_preview.models import OpenShiftManagedClusterMonitorProfile
+from azure.mgmt.containerservice.v2019_10_27_preview.models import OpenShiftAPIProperties
```

Comment:
Per https://github.com/Azure/sdk-release-request/issues/46 this should be available in the latest release (8.3.0).

Reply:
Seems to be missing the right models, I've pinged the release manager on the ticket.